testscommon/hasherMock.go

```diff
@@ -1,4 +1,4 @@
-package mock
+package testscommon
```

Comment:
just a thought: moving these mocks to elrond-crypto-go make sense?

Reply:
not quite, you do not want to depend on the plugins at test level